account_invoice_som/wizard/wizard_payment_order_add_invoices.py

```diff
@@ -120,6 +123,8 @@ def show_job_groups_progress(self, cursor, uid, ids, context=None):
         'invoice_type': fields.selection([(False, '')]+ INVOICE_TYPES, _('Tipus')),
         'fiscal_position': fields.many2one('account.fiscal.position', 'Posició Fiscal'),
         'payment_type': fields.many2one('payment.type', 'Tipus de pagament'),
+        'allow_grouped': fields.boolean('Permetre agrupacions',
+            help='Activar aquesta opció admet factures agrupades')
```

Comment:
```suggestion
            help='Activar aquesta opció admet factures agrupades'),
```

Afegir una coma final per a que si s'afegeix un camp en un futur, no surti aquest com a modificat..